protected/models/CurationLog.php

```diff
@@ -109,7 +109,7 @@ public static function makeNewInstanceForDatasetBy(int $id, string $creator): Cu
      */
     public static function createCurationLogEntry(int $id, string $fileName): bool
     {
-        $curationlog = self::makeNewInstanceForCurationLogBy($id, "System");
+        $curationlog = self::makeNewInstanceForCurationLogBy($id, Yii::app()->user->getFirstName() . " " . Yii::app()->user->getLastName());
```

Comment:
just take into account that firstName might be null or empty so you could end up with extra spaces when building the full name